internal/commands/list.go

```diff
@@ -22,55 +26,78 @@ import (
 var (
 	listColumns = []struct {
```

Comment:
I didn't took time to create a formatter for app ls command, but we should for homogeneity. Will look into this in a future PR.